src/main/java/com/revature/personalfinance/repo/IAccountRepo.java

```diff
@@ -9,8 +9,9 @@
 
     List<Account> getAccountsByUserId(int userId);
 
-    //Account updateAccount(Account account);
+//    Account updateAccount(Account account);
```

Comment:
Why the change why the commented code?

Reply:
It was something we were intended to use, but was not needed in the end. Not sure why the file got committed. I tried a bunch of things to remove it from the PR, but I just ended up reverting the changes.